acceptance/acceptance_test.go

```diff
@@ -678,44 +678,59 @@ func testAcceptance(
 						t.Log("app is runnable")
 						assertMockAppRunsWithOutput(t, repoName, "Launch Dep Contents", "Cached Dep Contents")
 
+						version, err := packVersion(packPath)
+						h.AssertNil(t, err)
+						packSemver = semver.MustParse(strings.TrimPrefix(strings.Split(version, " ")[0], "v"))
+						packSupport := packSemver.GreaterThan(semver.MustParse("0.10.0")) || packSemver.Equal(semver.MustParse("0.0.0"))
+
 						t.Log("restores the cache")
-						if lifecycleDescriptor.Info.Version.LessThan(semver.MustParse("0.6.0")) {
-							h.AssertContainsMatch(t, output, `(?i)\[restorer] restoring cached layer 'simple/layers:cached-launch-layer'`)
-							h.AssertContainsMatch(t, output, `(?i)\[analyzer] using cached launch layer 'simple/layers:cached-launch-layer'`)
-						} else {
+						if semver.MustParse(lifecycleDescriptor.API.PlatformVersion.String()).LessThan(semver.MustParse("0.3")) || !packSupport {
```

Comment:
I wonder if there would be value in introducing a new variable such as `lifecycleSupportsCreator` to make this more readable.

Reply:
Sure! Will do.